markdown/usage/data_guidelines.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Data Management Guidelines
+subtitle:
+---
+
+## Data management
```

Comment:
A general introduction here would be nice - I think what @mahesh-panchal suggests is a good idea: spelling out why data management and planning is important, and what value it gives.

Reply:
What about:

Research councils are recently recognizing the importance of managing data from research. Some now request detailed Data Management Plans (DMP) as part of the grant application. Guidance is provided below to aid the creation of DMP's, estimate resources needed by nf-core workflows, and how to share the resulting data.

Reply:
Or maybe Research funding agencies instead of research councils.

Reply:
```suggestion
## Data management
Funding agencies are recognizing the importance of research data management, and some are now requesting a Data Management Plan (DMP) as part of grant applications. Guidance is provided below to aid the creation of DMPs, estimate resources needed by nf-core pipelines and how to share the resulting data. 
```

Reply:
```suggestion
## Data management
Funding agencies are recognizing the importance of research data management and some now request detailed Data Management Plans (DMP) as part of the grant application. Guidance is provided below to aid the creation of DMPs, estimate resources needed by nf-core workflows, and how to share the resulting data.
```